docker-compose.yml

```diff
@@ -75,6 +75,7 @@ services:
         ET_URL: ${ET_URL}
         PRODUCT_DEF_URL: ${PRODUCT_DEF_URL}
         DASHBOARD_URL: ${DASHBOARD_URL}
+        BBSYNC_SYNC_TO_BZ: ${BBSYNC_SYNC_TO_BZ}
```

Comment:
requires an update to the operational repo too

Reply:
both PSI and MP+

Reply:
Good point. However do you think we still need PSI part here? I mean we only plan to enable Bugzilla backwards sync in the stage now and it is already in MP+ while the production enabling will happen maybe in late Q2 when we should already have everything in MP+.

Reply:
https://gitlab.corp.redhat.com/product-security/dev/osidb-ops/-/merge_requests/22

Reply:
I assume if the env variable isn't present in PSI that it will be treated as false so LGTM, good catch